### DISCUSSION TITLE
Plugin suggestion: Flinching attack timer

### DISCUSSION THREAD

**Discussion by Steierer420** [runelite/runelite#16758] [Ideas]
Hello,

My suggestion is to make a plugin, that tells you when the npc is ready to be attacked again while flinching. Inside the plugin settings you can set the attack speed of the npc since the duration depends on attack speed of the npc.

The formula of flinching is the following: [attackspeed / 2] + 8 ticks

Got the information of the wiki: https://oldschool.runescape.wiki/w/Flinching

I came up with that idea after 260 KQ kills lol.

I wish you all a nice weekend,

Markus

**Comment by Nightfirecat** [comment]
That's against [Jagex's Third Party Client Guidelines](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1), so we can't add it.

> Helps you to know when to 'flinch' your opponent
